docs/usermanuals/tr/data/croissant-card.md

```
@@ -0,0 +1,132 @@
+
+:::warn
+**Card'ı düzenleyip sonra audit'i yeniden çalıştırma.** `forgelm audit`'i yeniden çalıştırmak dosyanın üzerine yazar. Ya son audit *sonrası* düzenleyin ya da düzenlenmiş card'ı kendi publish-step script'inizle ileri besleyin.
+:::
+
+:::tip
+Hugging Face yayını için card'ı dataset repo'sunda `croissant.json` olarak kaydedin (HF orada bekler). Basit bir `jq '.croissant' data_audit_report.json > croissant.json` işi görür.
+:::
+
+## Bakınız
+
+- [Veri Seti Denetimi](#/data/audit) — `--croissant`'ın çağrıldığı yer.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)